web/src/components/forms/dynamic-form/const.ts

```diff
@@ -0,0 +1,33 @@
+export const K8S_UI_SCHEMA = {
```

Comment:
nit / ok for a follow-up: constants are not all-caps (see previously https://github.com/netobserv/network-observability-console-plugin/pull/541)